views/show.ejs

```diff
@@ -0,0 +1,33 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+	<h1> <%= pageTitle %> </h1>
+	<p>
+
+<table style="width:50%">
+
+	<% Object.keys(mission).forEach(function(key) { %>
```

Comment:
Why Object.keys.forEach? It works so it's fine. Just wondering.  You could also use `for in`

Reply:
I found both options, but since it was an object with a key I used that one and since that one worked I didn't try to use for in.